### ISSUE TITLE
own-props-must-be-private for private properties

### ISSUE THREAD

**Issue by runarberg** [ionic-team/stencil-eslint#26]
The `own-props-must-be-private` persists even for private properties. E.g.

```ts
@Component(/* ... */)
class MyComponent {
  #privateProperty = "secret";  // <-- false positive
}
```

**Comment by StevenMeyer** [comment]
I need this in my project, so I've made a PR. Hopefully the maintainers will come back soon and update 😄
